docs/reference-sequence-import.md

```diff
@@ -0,0 +1,124 @@
+# Importing Reference Sequence
+
+ This document shows the flow of how different types of ReferenceSequences are extracted and imported.
+
+## Types of Reference Sequence to import
+   
+   * ASEMBLY
+   * GENE
+   * TRANSCRIPTOME
+ 
+## Retrieving Reference Sequence Accessions
+
+ SRA Analysis records can contain a reference sequence in either of the three analysis categories:
+   * REFERENCE_ALIGNMENT
+   * SEQUENCE_VARIATION 
+   * PROCESSED_READS. 
+   
+ It is guaranteed that each analysis contains at most one of these three types.
+ Example xml below which is parsed to java object ,
+ 
+ ```
+ <SEQUENCE_VARIATION>
+     <ASSEMBLY>
+         <STANDARD accession="GCA_000001405.1"/>
+     </ASSEMBLY>
+     <SEQUENCE accession="GBRU01" label="GBRU01"/>
+     <SEQUENCE accession="JQ739518.1" label="JQ739518.1"/>
+     <SEQUENCE accession="CM000665.1" label="chr3"/>
+     [...]
+ </SEQUENCE_VARIATION>
+```
+
+ ##### Retrieving ASSEMBLY accession
+  The Assembly type reference sequence accession is represented in below part of the xml.
+  ``` 
+  <ASSEMBLY>
+          <STANDARD accession="GCA_000001405.1"/>
+  </ASSEMBLY>
+  ```
+  
+ ##### Retrieving GENE OR TRANSCRIPTOME accession
+  The GENE or TRANSCRIPTOME type reference sequence accession is represented in below part of the xml.
+   ```
+        <SEQUENCE accession="GBRU01" label="GBRU01"/>
+        <SEQUENCE accession="JQ739518.1" label="JQ739518.1"/>
+   ```
+   
+ ## Retrieving Reference Sequence Xmls from accessions
+ 
+ The reference sequence xml is retrieved via ENA API like other objects
+ 
+ ##### ASSEMBLY XML
+  The assembly accession returns an assembly xml which can be parsed via ASSEMBLYDocument factory parser.
+   The patches are calculated only for GRch| GRcm from the alias name. A reference sequence is created by parsing the
+    accession , name , patch and taxon details.
+   Example of assembly xml below,
+   ```
+   <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
XML examples don't need extra indentation as well (in this example, every line has 3 extra spaces in front of it)

Reply:
fixed in f3f80b9